src/dom/set.js

```diff
@@ -29,7 +31,10 @@ const setProps = {
 		return bind(subject, val, {once: true});
 	},
 
-	// contents,
+	// Why not simply contents? To avoid Uncaught ReferenceError: Cannot access "contents" before initialization.
+	contents: function (subject, val) {
```

Comment:
I would use `...args` instead of specifying the arguments, in case its signature changes.

Reply:
Thanks.